vnext/Shared/OInstance.h

```diff
@@ -61,6 +63,10 @@ class InstanceImpl final : public InstanceWrapper, private ::std::enable_shared_
     return m_innerInstance;
   }
 
+  virtual const std::shared_ptr<RuntimeScheduler> &GetRuntimeScheduler() const noexcept override {
+    return m_runtimeScheduler;
+  }
```

Comment:
Depending on what you decide about passing the ReactPropertyBag through, you can probably remove this property from InstanceWrapper, etc.